readme.md

```diff
@@ -55,16 +89,18 @@ Post upload, the bucket contents should look similar to this:
 2019-09-30 14:14:36       1244 scripts/ssm
 ```
 
-It is recommended that this template be deployed in the same subnet as your EMR cluster.
```

Comment:
I don't love this either, but some form needs to exist in this section.   Deploying in a subnet with a separate routing table attached will prevent connectivity between the notebook and livy.